packages/runtime/id-compressor/src/idCompressor.ts

```diff
@@ -260,7 +260,9 @@ export class IdCompressor implements IIdCompressor, IIdCompressorCore {
 		if (lastCluster === undefined) {
 			// This is the first cluster in the session space
 			if (rangeBaseLocal !== -1) {
-				throw new Error("Ranges finalized out of order.");
+				throw new Error(
```

Comment:
Interpolating strings in telemetry is usually a worse experience than

```typescript
import { LoggingError } from "@fluidframework/telemetry-utils/internal";

// at throw
throw new LoggingError("Ranges finalized out of order", { expectedStart: -1, actualStart: rangeBaseLocal });
```

and similarly below.
```

Reply:
maybe worth a one-line helper at the top of the file :)

Reply:
done!